docs/source-app/workflows/ssh/index.rst

```diff
@@ -0,0 +1,83 @@
+
+#################################
+Access your Apps via SSH
+#################################
+
+**Audience:** Users who want to debug their Flow and Work in a the runtime environment.
+
+----
+
```

Comment:
Add motivation-

When running your app on the cloud, it could be useful to SSH to cloud machines so you can debug them on your local machine as if they are running locally:
• Inspect files
• Inspect environment
• Monitor resources
etc